src/test/java/uk/gov/ons/census/action/builders/QidUacBuilderTest.java

```diff
@@ -2,17 +2,14 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.Assert.assertEquals;
-import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.ArgumentMatchers.*;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
-import static uk.gov.ons.census.action.builders.QidUacBuilder.HOUSEHOLD_INITIAL_CONTACT_QUESTIONNAIRE_TREATMENT_CODE_PREFIX;
-import static uk.gov.ons.census.action.builders.QidUacBuilder.WALES_TREATMENT_CODE_SUFFIX;
+import static org.postgresql.hostchooser.HostRequirement.any;
```

Comment:
dodgy import. These are not the `any` droids you are looking for

Reply:
You need the mockito matchers `any` instead.

Reply:
Are it's wildcard importing the argument matcher so this line can just be deleted